addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -1395,6 +1329,9 @@ def test_out_invoice_line_onchange_analytic(self):
         with move_form.invoice_line_ids.edit(0) as line_form:
             line_form.analytic_account_id = self.env['account.analytic.account']
             line_form.analytic_tag_ids.clear()
+        with move_form.invoice_line_ids.edit(1) as line_form:
+            line_form.analytic_account_id = self.env['account.analytic.account']
+            line_form.analytic_tag_ids.clear()
```

Comment:
The first line was computed in the onchange because done in the form emulator.
The second wasn't done before because never edited, but done now because it's computed at creation.